src/components/Thread.vue

```diff
@@ -501,8 +501,21 @@ export default {
 	display: none;
 }
 .user-bubble__wrapper {
-	margin-right: 4px;
+	margin-right: var(--default-grid-baseline);
+	background-color: var(--color-background-dark);
+	border-radius: var(--border-radius-large);
+	padding: calc(var(--default-grid-baseline) / 2);
+}
+
+.v-popper__popper--shown .user-bubble__wrapper {
+	margin-right: 0 !important;
 }
+
+.user-bubble__content {
+	height: unset !important;
```

Comment:
According to https://nextcloud-vue-components.netlify.app/#/Components/NcUserBubble there is also a size prop for the user bubble